monitoring/README.md

```diff
@@ -193,10 +199,14 @@ applications are available at these locations by default:
 * Prometheus - Port 31090 `http://master-node.yourcluster.example.com:31090`
 * AlertManager - Port 31091 `http://master-node.yourcluster.example.com:31091`
 
-The default Grafana admin user is `admin`. Unless otherwise specified,
-the default password is randomly generated. The value is logged only
-during the initial deployment of the monitoring components and not on
-an upgrade.
+The default Grafana admin user is `admin`. Unless you used the `GRAFANA_ADMIN_PASSWORD` environment variable (either in the `user.env` file or on the command line) to specify a default password during deployment, the default password is randomly generated and displayed during deployment. 
+
+If you want to change the password, issue this command:
+```bash
+kubectl exec -n <monitoring_namespace> <grafana_pod> -c grafana -- bin/grafana-cli admin reset-admin-password myNewPassword
+```
```

Comment:
@BryanEllington, is this the correct command to change the Grafana password?

Reply:
Yes, that is correct